packages/manager/tools/webpack-config/src/alias/iam.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  module: '@ovh-ux/manager-iam',
```

Comment:
**issue(blocking):** Shared webpack configuration mustn't be coupled to a module nor feature business logic.